src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/MicrosoftIdentityPlatform/MicrosoftIdentityPlatformApplicationManager.cs

```diff
@@ -37,19 +40,19 @@ public class MicrosoftIdentityPlatformApplicationManager
                 Description = applicationParameters.Description
             };
 
-            if (applicationParameters.IsWebApi)
+            if (applicationParameters.IsWebApi.HasValue && applicationParameters.IsWebApi.Value)
```

Comment:
GetValueOrDefault() should give you the same behavior as .HasValue && .Value, since the default is 'false'

Reply:
Oh good find that'll help a bunch